src/Fields/Fields.jl

```diff
@@ -332,4 +332,20 @@ function level(field::FaceExtrudedFiniteDifferenceField, v::PlusHalf)
     Field(data, hspace)
 end
 
+"""
+    set!(field::Field, f::Field)
+    set!(field::Field, f::Function)
+
+Assign values of `f`, 
+which may be a field or a
+function, to field `field`.
```

Comment:
Docs should explain what arguments `f` should take.

Also should `f` be the first arg, so we can use `do` notation?

Docstring needs to be added to documenter as well